libmscore/bracket.cpp

```diff
@@ -243,11 +243,11 @@ void Bracket::draw(QPainter* painter) const
                   else {
                         qreal h        = 2 * h2;
                         qreal _spatium = spatium();
-                        qreal mag      = h / (4 *_spatium);
+                        qreal mag      = h / 100;
```

Comment:
Leaves `_spatium` unused and so causes a compiler warning

Reply:
True! thanks